doc/releases/changelog-0.31.0.md

```diff
@@ -403,24 +727,24 @@
   of `mult` that is supported.
   [(#4058)](https://github.com/PennyLaneAI/pennylane/pull/4058)
 
-<h3>Bug fixes 🐛</h3>
+<h3>Bug Fixes 🐛</h3>
 
-* Fixes the matrix of `SProd` when the coefficient is tensorflow and the target matrix is not `complex128`.
-  [(#4249)](https://github.com/PennyLaneAI/pennylane/pull/4249)
-
-* Fixes adjoint jacobian results with `grad_on_execution=False` in the JAX-JIT interface.
+* Fixed adjoint jacobian results with `grad_on_execution=False` in the JAX-JIT interface.
   [(4217)](https://github.com/PennyLaneAI/pennylane/pull/4217)
 
-* Fixes a bug where `stoch_pulse_grad` would ignore prefactors of rescaled Pauli words in the
+* Fixed the matrix of `SProd` when the coefficient is tensorflow and the target matrix is not `complex128`.
+  [(#4249)](https://github.com/PennyLaneAI/pennylane/pull/4249)
+ 
+* Fixed a bug where `stoch_pulse_grad` would ignore prefactors of rescaled Pauli words in the
   generating terms of a pulse Hamiltonian.
   [(4156)](https://github.com/PennyLaneAI/pennylane/pull/4156)
   
-* Fixes a bug where the wire ordering of the `wires` argument to `qml.density_matrix`
+* Fixed a bug where the wire ordering of the `wires` argument to `qml.density_matrix`
   was not taken into account.
   [(#4072)](https://github.com/PennyLaneAI/pennylane/pull/4072)
 
-* Removes a patch in `interfaces/autograd.py` that checks for the `strawberryfields.gbs` device.  That device
-  is pinned to PennyLane <= v0.29.0, so that patch is no longer necessary.
+* A patch in `interfaces/autograd.py` that checks for the `strawberryfields.gbs` device has been removed. 
+  That device is pinned to PennyLane <= v0.29.0, so that patch is no longer necessary.
```

Comment:
```suggestion
  That device is pinned to PennyLane <= v0.29.0, so that patch is no longer necessary.
  [(#4089)](https://github.com/PennyLaneAI/pennylane/pull/4089)
```